packages/mui-base/src/useSlider/useSlider.ts

```diff
@@ -224,6 +224,12 @@ export default function useSlider(parameters: UseSliderParameters): UseSliderRet
     name: 'Slider',
   });
 
+  const isValueUnchanged = (newValue: number | Array<number>): boolean => {
```

Comment:
```suggestion
  const areValuesEqual = (oldValue: number | Array<number>, newValue: number | Array<number>): boolean => {
```

Let's rename the util, it has negation in the name, and we use another negation when invoking. Also, let's change the signature and move it outside of the component.

Reply:
Done